### PR TITLE
[risk=low][no ticket] Bump types/lodash to latest and refactor a few `cond()`

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/pages/admin/admin-workspace.tsx [hunk 1]
```diff
@@ -167,21 +166,20 @@ const NameCell = (props: NameCellProps) => {
...
     filename.endsWith(NOTEBOOKS_SUFFIX);
-  const isTooLargeNotebook = () =>
-    isNotebook() && file.sizeInBytes > MAX_NOTEBOOK_READ_SIZE_BYTES;
+  const isTooLargeNotebook =
+    isNotebook && file.sizeInBytes > MAX_NOTEBOOK_READ_SIZE_BYTES;
 
   // if (tooLarge()) fileTooLarge();
   // else if (isNotebook()) fileWithPreviewButton();
   // else filenameSpan();
-  const requiredDummyParameter = undefined;
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-workspace.tsx:179]: My understanding as to what happened here:

originally the type signatures were incorrect for this use of `fp.cond()` so a dummy parameter was required to compile.  A new version corrects this, which causes this code to no longer compile.

We can make this moot however, as we have our own `cond()` which is appropriate when there's no argument needed.

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 2]
```diff
@@ -1745,7 +1745,10 @@ const RuntimePanel = fp.flow(
     const { enableGpu, enablePersistentDisk } = serverConfigStore.get().config;
 
     const initialPanelContent = fp.cond([
-      [([b, ,]) => b === BillingStatus.INACTIVE, () => PanelContent.Disabled],
+      [
+        ([b, _, _2]) => b === BillingStatus.INACTIVE,
```
Comment (jmthibault79) [inline ui/src/app/pages/analysis/runtime-panel.tsx:1749]: we must now specify all parameters

**Comment by jmthibault79** [comment]
7 e2e failures, all notebooks.  Likely the 504 issue.

File: ui/yarn.lock [hunk 3]
```diff
@@ -1940,10 +1940,10 @@
   resolved "https://registry.yarnpkg.com/@types/json5/-/json5-0.0.29.tgz#ee28707ae94e11d2b827bcbe5270bcea7f3e71ee"
   integrity sha1-7ihweulOEdK4J7y+UnC86n8+ce4=
 
-"@types/lodash@4.14.179":
```
Comment (NehaBroad) [inline ui/yarn.lock:1943]: do we merge yarn.lock files?

Comment (jmthibault79) [reply ui/yarn.lock:1943]: what do you mean?

**Review by evrii — APPROVED** [review]
Looks good to me. I was able to run locally. I could get the changes to work for admin-workspace, but I don't know of a good way to test the changes from the runtime panel.

**Review by NehaBroad — APPROVED** [review]

**Comment by jmthibault79** [comment]
The runtime panel change was not meaningful - it's only what is required to compile with the version update

**Pull request merged**